shared/src/question-generators/example/example.ts

```diff
@@ -28,7 +28,7 @@ function generateWrongAnswers(
   random: Random,
   correctAnswer: number,
 ): Array<string> {
-  let wrongAnswers = [
+  const wrongAnswers = [
```

Comment:
Just as a side note: this change is correct but does not belong into this branch/commit. Just leave it in for now but in general try to keep the branches/commits focused.

Reply:
Okay, I will write it down in the future and then do those commits separately 👍